Document `wrapUseRoutes` from React Router 6 instrumentation. (#5475)
getsentry/sentry-docs: src/platforms/javascript/guides/react/configuration/integrations/react-router.mdx

To use React Router v6 with Sentry:

Initialize `Sentry.reactRouterV6Instrumentation` as your routing instrumentation and provide the functions it needs to enable performance tracing:

    - `useEffect` hook from `react`
    - `useLocation` and `useNavigationType` hooks from `react-router-dom`
    - `createRoutesFromChildren` and `matchRoutes` functions from `react-router-dom` or `react-router` packages.
    

> ### Usage with `<Routes />` component

Reviewer: ```suggestion
### Usage With `<Routes />` Component
```

> If you use `<Routes />` component from `react-router-dom` to define your routes, wrap [`Routes`](https://reactrouter.com/docs/en/v6/api#routes-and-route) using `Sentry.withSentryReactRouterV6Routing` to create a higher order component, which will enable Sentry to reach your router context, as in the example below:

Reviewer: Sentence is a bit long

```suggestion
If you use the `<Routes />` component from `react-router-dom` to define your routes, wrap [`Routes`](https://reactrouter.com/docs/en/v6/api#routes-and-route) using `Sentry.withSentryReactRouterV6Routing`. This creates a higher order component, which will enable Sentry to reach your router context, as in the example below:
```

);
```

> ### Usage with `useRoutes` hook

Reviewer: ```suggestion
### Usage With `useRoutes` Hook
```

> _(Available in version 7.12.0 and above)_

Reviewer 2: ```suggestion
_(Available in version 7.12.1 and above)_
```

If you specify your route definitions as an object to the [`useRoutes` hook](https://reactrouter.com/en/main/hooks/use-routes), use `Sentry.wrapUseRoutes` to create a patched `useRoutes` hook that instruments your routes with Sentry.

<Alert level="warning">

> `wrapUseRoutes` should be called outside of a React component, as in the example below. It's also recommended to assign the wrapped hook to a variable name starting with `use`, in accordance with [React documentation](https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook).

Reviewer: ```suggestion
`wrapUseRoutes` should be called outside of a React component, as in the example below. It's also recommended that you assign the wrapped hook to a variable name starting with `use`, as per the [React documentation](https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook).
```
